cp-main.go

```diff
@@ -45,6 +52,14 @@ var (
 			Name:  "recursive, r",
 			Usage: "Copy recursively.",
 		},
+		cli.BoolFlag{
+			Name:  "skip-copy",
```

Comment:
I think fake should be added as well. Maybe `skip-copy` is being named wrong, but what I want with this option is to skip the initial copy and scan, and only start monitoring.